_data/releases/1.9/1.9.0.Alpha2.yml

```
@@ -2,4 +2,4 @@ date: 2022-02-09
 version: "1.9.0.Alpha2"
 stable: false
 summary: Redis for Debezium Server improvements; Oracle 21c support; Configurable KAFKA_QUERY_TIMEOUT
```

```suggestion
summary: Redis for Debezium Server improvements; Oracle 21c support; Configurable `kafka.query.timeout.ms`
```

Let make this suggestion as well; as I think it conveys what is configurable more formerly than the name of the variable in the code as I had originally added to the release YAML.